scripts/rebalance-queue-masters

```diff
@@ -37,7 +37,7 @@ declare -i node_count=0
 sync_queue()
 {
     echo -n "$(now) [INFO] "
-    rabbitmqctl sync_queue -p "$vhost" "$1"
+    rabbitmqctl sync_queue -p "$vhost" "$1" || echo "ignoring error"
```

Comment:
Should we make it more specific what error is being ignored? E.g. "ignoring an error from sync_queue"?

Reply:
Yes, that's more helpful. I'll change it.

Reply:
I removed this message with the latest change.